app/assets/javascripts/discourse/app/lib/avatar-flair.js

```diff
@@ -0,0 +1,61 @@
+let _autoGroupFlair, _noAutoFlair;
```

Comment:
We need a way to reset local module variables like this, otherwise in tests they keep state if the test initializes them. I recommend adding a `resetFlair` method that is called after each test.